yesod-test/test/main.hs

```diff
@@ -201,6 +201,9 @@ main = hspec $ do
             yit "finding html" $ do
                 get ("/html" :: Text)
                 statusIs 200
+                htmlAnyContainPreEscaped "meta" "content=\"site description\""
+                htmlAllContainPreEscaped "meta"  "content=\"site description"
+                htmlNoneContainPreEscaped "meta" "foobar"
```

Comment:
This checks absence in the same was as the test for `htmlNoneContain`, I think it'd be better with a test case which illustrate the difference between the two functions.